scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -170,6 +170,7 @@ class FossId(
         user = fossIdScannerOptions["user"]
             ?: throw IllegalArgumentException("No FossId User configuration found.")
         packageNamespaceFilter = fossIdScannerOptions["packageNamespaceFilter"].orEmpty()
+        packageAuthorsFilter = fossIdScannerOptions["packageAuthorsFilter"].orEmpty()
```

Comment:
BTW, I believe all of these FOSSID-specific options also deserve some extra docs, explaining what exactly they're good for (what's the use-case). Can I also expect another commit for that?

Reply:
And the commit message should make more clear that this is a generic, scanner-specific option, not a top-level `ort.conf` entry.